{{cookiecutter.project_slug}}/server/{{cookiecutter.project_slug}}/settings.py

```diff
@@ -357,7 +357,6 @@
     "handlers": {
         "console": {
             "level": "INFO",
-            "filters": ["require_debug_true"],
```

Comment:
The only possible controversial change here.
By default we were effectively turning off logs in production. Some things would get logged, but there would be missing details.
In my mind a project should start with very verbose logs in prod. If the devs find that too noisy they can turn them off later when that becomes a problem.

Reply:
Ah, this must have been mangled along the way. The "console" handler in Django's default logging requires DEBUG=True, because it has a separate "mail admins" config for production when DEBUG=False. You can see here: https://github.com/django/django/blob/main/django/utils/log.py#L18

Long ago, I overloaded the default logging config here in settings.py so that we could always have the 'django.requests' and 'django.tasks' sub-loggers below and to also make some customizations to the formatters. This is definitely worth a revisit now.

Also, I still see logs in production for apps using this config (like Squirrel), so that's interesting...

Reply:
yea I made this change a bit ago so I kinda forgot the details. I do recall logs still being in prod, but they weren't full detail. Definitely not using the verbose formatter, so you wouldn't get things like line numbers and I think maybe INFO logs weren't showing up(?)

Interesting context. Honestly I also don't love a change like this because it's not obvious how a future dev is to know to add `require_debug_true` as a filter should they want to change the behavior. I think many devs would end up googling their own solution.

Reply:
OK, this is probably worth a GH issue someone can tackle to test and tweak this in more detail to something we like, but for now I'm on board with this change.

For now it might also be good to add a link to Django's logging docs, in case anyone is wondering what the heck is going on here.